README.md

```diff
@@ -61,6 +61,9 @@ spec:
   deschedulingIntervalSeconds: 1800
   profiles:
   - AffinityAndTaints
+  - LifecycleAndUtilization
+  profileCustomizations:
+    podLifetime: 5m
```

Comment:
@damemi above does not work, it throws us an error which reads as below, so the value should be only integer

spec.profileCustomizations.podLifetime: Invalid value: "string": spec.profileCustomizations.podLifetime in body must be of type integer: "string"

Reply:
Opened a new PR to address this: https://github.com/openshift/cluster-kube-descheduler-operator/pull/211